### PR TITLE
Drivetrain Constants Matched to Robot and Tested

### PR DESCRIPTION
**Pull request by dstarner**
* Updated Drivetrain constants for the CAN IDs to match what's on the robot
* Allow for inverting the move speed value from the controller
* More comments & documentation

### CODE REVIEW

**Review by dennisgove — APPROVED** [review]
lgtm

File: robot/src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -33,6 +34,7 @@
         public static final int kRightStickY = 5;
         public static final int kRightTrigger = 3;
 
+        public static final boolean kInvertMoveSpeed = true;
```
Comment (zhiquanyeo) [inline robot/src/main/java/frc/robot/Constants.java:37]: This shouldn't be a property of the Gamepad, but instead of the drivetrain

**Comment by zhiquanyeo** [comment]
d'oh! Dennis merged it before i put in my comment :P

**Comment by dennisgove** [comment]
Collaboration!  :)

File: robot/src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (dennisgove) [reply robot/src/main/java/frc/robot/Constants.java:37]: I think this depends on which is providing the inverted value. Is the controller providing a negative value to move forward or is the drivetrain receiving a negative value and moving forward.

If it's the former then negating in the drivetrain will become problematic for autonomous commands. If it's the latter then I agree negating in the drivetrain is correct.

Comment (zhiquanyeo) [reply robot/src/main/java/frc/robot/Constants.java:37]: I think that historically negative controller values = forward/counter clockwise turn. I'm open to whichever, as long as we're consistent. (Also I think some autonomous pathfinding libs have the same convention of -ve = forward)

Comment (dstarner) [reply robot/src/main/java/frc/robot/Constants.java:37]: I can add the invert to both the game pad and the drivetrain, and we can play with the values when we do autonomous driving

Comment (zhiquanyeo) [reply robot/src/main/java/frc/robot/Constants.java:37]: If we're going down that route, then can I suggest that we add flags for inverting every axis? And name the axis accordingly (e.g. leftX, leftY, etc)

**Pull request merged**